src/components/GoodsList/GoodsList.tsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import './GoodsList.scss';
+import '../../styles/button.scss';
+
+interface Props {
+  goods: string[],
+}
+
+interface State {
+  goods: string[],
+  sortBy: string,
+}
+
+export class GoodsList extends React.Component<Props> {
+  state = {
+    goods: [...this.props.goods],
+    sortBy: 'alphab',
+  };
+
+  reverse = () => (
+    this.setState((state: State) => ({
+      goods: state.goods.reverse(),
+    }))
+  );
+
+  sortAlphab = () => {
+    this.setState({
+      sortBy: 'alphab',
+    });
+
+    this.sort();
+  };
+
+  length = () => {
```

Comment:
I think "sortLength" method name would be better

Reply:
yes, function names should start with verbs to tell what they do